docs/api.md

```diff
@@ -26,6 +26,7 @@
 .. autofunction:: scores.probability.crps_cdf_brier_decomposition
 .. autofunction:: scores.probability.murphy_score
 .. autofunction:: scores.probability.murphy_thetas
+.. autofunction:: scores.probability.quantile_score
```

Comment:
This should go in `continuous` rather than `probability` (with scores such as MSE, MAE etc).

You'll have the update the location of the rest of the new files that you've added.

Reply:
Thanks. It has been fixed.